tardis/montecarlo/tests/Makefile

```diff
@@ -0,0 +1,22 @@
+all: test_cmontecarlo
+
+rk_isaac.o: ../src/randomkit/rk_isaac.c
+	g++ -fpermissive -c ../src/randomkit/rk_isaac.c
```

Comment:
try to get the compiler from setup.py. g++ is not always available (Macs mostly use clang)